RELEASE-NOTES.txt

```diff
@@ -7,7 +7,6 @@
 * [*] Quick Start: The "Get to know the WordPress app" card has a fresh new look [#18688, #18747]
 * [*] Block Editor: A11y: Improve text read by screen readers for BottomSheetSelectControl [https://github.com/WordPress/gutenberg/pull/41036]
 * [*] Block Editor: Add 'Insert from URL' option to Image block [https://github.com/WordPress/gutenberg/pull/40334]
-* [*] Quick Start: The "Get to know the WordPress app" card has a fresh new look [#18688]
```

Comment:
I forgot to update this file when making https://github.com/wordpress-mobile/WordPress-iOS/commit/4ec2364eca97b11dd5ae237e3e71494f8c58833b 🤦‍♂️ Thankfully, @tiagomar and @momo-ozawa were onto it.

Reply:
👍